simulation_code/simulations.jl

```diff
@@ -274,8 +297,12 @@ end
         # make choice
         ce_Uit = certainty_equivalent(alpha, mu_Uit, Sigma_Uit)
         choice_set = [n for n in Nset if n ∉ C_iT]
+        
         c_it = 0
-        if rand() < epsilon
+        if use_thompson
+            c_it = thompson_sampling(mu_U_i, Sigma_U_i)
+            @show c_it
```

Comment:
this is probably useful for debugging but let's kill this before merge?

Reply:
resolved in 80e579c